tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -128,6 +128,30 @@ When(
   }
 )
 
+When(
+  /^"([^"]*)" (copies|moves) the following resources to "([^"]*)" at once using (keyboard|drag-drop|drag-drop-breadcrumb|dropdown-menu|batch-action)$/,
+  async function (
+    this: World,
+    stepUser: string,
+    actionType: string,
+    newLocation: string,
+    method: string,
+    stepTable: DataTable
+  ): Promise<void> {
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const resourceObject = new objects.applicationFiles.Resource({ page })
+
+    const resources = [].concat(...stepTable.rows())
```

Comment:
```suggestion
    const resources = stepTable.rows()
```

Reply:
`stepTable.rows()` will return two-dimensional array. For easier processing, converted the array into one-dimensional array.